opentelemetry-sdk/src/opentelemetry/sdk/environment_variables.py

```diff
@@ -335,6 +335,15 @@
 A scheme of https indicates a secure connection and takes precedence over this configuration setting.
 """
 
```

Comment:
Do we need `OTEL_EXPORTER_OTLP_METRICS_CLIENT_KEY `, `OTEL_EXPORTER_OTLP_METRICS_CLIENT_CERTIFICATE `

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/c7ce6098cb30d5e3fe81dcea18a7a51479e3816b/specification/protocol/exporter.md#configuration-options

Reply:
Added them